docs/index.md

```diff
@@ -3,9 +3,9 @@ Gaffer 2
 
 *These Docs are a work in progress for Gaffer v2, [the docs for Gaffer v1 can be found here](../v1docs/)*.
 
-For information on Deprecated classes which have been removed from Gaffer with version 2, [please see this page](deprecations.md).
+For information and migration steps for Deprecated classes which have been removed in Gaffer 2 alpha 1, [please see the deprecations page](deprecations.md).
 
-For a summary of changes in Gaffer 2, [please see the changes page](changes.md).
+For a summary of the current Gaffer 2 alpha roadmap, including migration steps, please see the [Gaffer 2 Changelist](changelist.md).
```

Comment:
If the name of the file changes, then the navbar link needs updating too - set in `mkdoc.yml`.